agent/consul/auto_config_endpoint.go

```diff
@@ -284,7 +285,12 @@ func (ac *AutoConfig) updateTLSSettingsInConfig(_ AutoConfigOptions, resp *pbaut
 	resp.Config.TLS.PreferServerCipherSuites = base.PreferServerCipherSuites
 
 	var err error
+	// FIXME: is the base.CipherSuites uint16 value exported or stored in
+	// memory remotely anywhere, or is this always passed as a string?
+	// This _might_ be okay regardless, as the underlying values are both
+	// IANA uint16 constant values.
 	resp.Config.TLS.CipherSuites, err = tlsutil.CipherString(base.CipherSuites)
+
```

Comment:
@i0rek or @mkeeler, would either of you know whether this underlying value is used over the wire anywhere, or whether string representations are always used for autoconfig?